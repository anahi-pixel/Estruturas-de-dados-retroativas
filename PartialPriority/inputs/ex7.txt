1 70 700
1 50 500
1 60 600
1 10 300
1 40 400
1 80 800
1 30 100
2 20
2 35
2 55
1 50 150
5
6
3 40
5
6
1 90 900
5
6
4 20
7
5
6
