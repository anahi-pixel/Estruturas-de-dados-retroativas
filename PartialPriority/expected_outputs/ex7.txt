400
5
500
4
500
5
1 Fila de prioridade no instante atual: 
300 500 600 700 800 900 
300
6
